### ISSUE TITLE
Insufficient color contrast for states of "I'm not a robot" Checkbox

### ISSUE THREAD

**Issue by BarrierbreakApp** [google/recaptcha#436]
Example URL: https://www.google.com/recaptcha/api2/demo

Background/foreground color combinations with an insufficient contrast ratio were found:
• The #C1C1C1 "I'm not a robot" checkbox on focus has a color contrast ratio of 1.7:1  with #F9F9F9 outer background.
• The #C1C1C1 "I'm not a robot" checkbox on focus has a color contrast ratio of 1.8:1  with #FFFFFF inner background.

Suggestion:
For all states of user interface controls, ensure that the color contrast for controls with their adjacent colors is 3:1.

![reCaptcha_4](https://user-images.githubusercontent.com/16647640/107737950-b0abc300-6d2b-11eb-809d-6c55729dc581.png)

**Comment by rowan-m** [comment]
These all look like automated issues for the widget unrelated to the PHP client library in this repo.
